data-serving/scripts/import-data/tsconfig.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
We should set up a base tsconfig and extend it for each ts app.

I'll do this in a separate PR -- just remarking here in case the thought occurred to anyone else.

Reply:
sgtm

Reply:
thanks for doing that ab